vs2015/activate.bat

```diff
@@ -21,8 +21,7 @@ if "%VSINSTALLDIR%" == "" (
 )
 
 if "%VSINSTALLDIR%" == "" (
-   ECHO "Did not find VS in registry or in VS140COMNTOOLS env var - exiting"
-   exit 1
+   ECHO "WARNING: Did not find VS in registry or in VS140COMNTOOLS env var - your compiler may not work"
```

Comment:
You probably want to `Exit 0` here rather than fall through.

Reply:
@dhirschfeld
I was told that the activate scripts are handled through the equivalent of `eval`, meaning any exit would abort the go. Will test this.

Reply:
I tested this - `exit 0` is as solidly broken as `exit 1`. The only reasonable path is not to have an exit.

Reply:
Right, I thought it was just the failed exit code but that makes sense. I'd suggest maybe a `goto :end` then (on phone so syntax may not be correct there)

Reply:
I made the adjustment - wish I had thought of that :)